iac/terraform/aws/cognito/cognito.tf

```diff
@@ -0,0 +1,73 @@
+locals {
+  from_email_address = "no-reply@${var.domain_name}"
```

Comment:
This limits the option of using a custom email address. Should be taken from terraform variable.

Reply:
Pass email address via terraform variable.
You will find it with this [commit](https://github.com/CraftsmenLtd/BloodConnect/commit/67d02235a3c4d0aeef23aeada28a414555c9de6c)